crates/turbopack-ecmascript/src/chunk/optimize.rs

```diff
@@ -269,11 +272,69 @@ async fn merge_duplicated_and_contained(
     Ok(())
 }
 
-async fn merge_by_size(chunks: &[EcmascriptChunkVc]) -> Result<Vec<EcmascriptChunkVc>> {
+async fn merge_to_limit(
+    chunks: Vec<(EcmascriptChunkVc, Option<ChunksVc>)>,
+    target_count: usize,
+) -> Result<Vec<EcmascriptChunkVc>> {
+    let mut remaining = chunks.len();
+    let mut chunks = chunks.into_iter().enumerate().collect::<Vec<_>>();
+    chunks.sort_by_key(|&(i, (_, k))| (k, i));
+    let mut chunks_by_source = chunks
+        .group_by(|(_, (_, a)), (_, (_, b))| a == b)
+        .map(|slice| {
+            (
+                slice[0].1 .1,
```

Comment:
Weird space
```suggestion
                slice[0].1.1,
```

Reply:
cargo fmt adds this. I guess it should not be a number literal